hazelcast/src/main/java/com/hazelcast/core/MultiMap.java

```diff
@@ -130,7 +130,18 @@
      * might be modifiable but it has no effect on the multimap.
      */
     Collection<V> remove(Object key);
-
+    /**
+     *  Deletes all the entries with the given key.
+     * <p/>
+     * <p><b>Warning:</b></p>
+     * This method uses <tt>hashCode</tt> and <tt>equals</tt> of the binary form of
+     * the <tt>key</tt>, not the actual implementations of <tt>hashCode</tt> and <tt>equals</tt>
+     * defined in the <tt>key</tt>'s class.
+     *
+     * @param key   the key of the entry to remove
+     * @returnv void
```

Comment:
I don't think `void` is documented, also typo in `@returnv`

Reply:
Thanks, changed.